core/src/main/java/org/jboss/gwt/elemento/core/builder/ElementBuilder.java

```diff
@@ -40,14 +40,17 @@
     }
 
     @Override
+    public E get() { return element; }
+
+    @Override
```

Comment:
I see you already have noticed but; I have renamed the getter of the TypedBuilder interface, which is a pretty generic interface that maybe we can use in the future to configure request or some other wrap-elemental2-type to improve type-safety utility. And, this should not break the code bc all elements builders extends from ElementBuilder which is an `IsElement` itself, so `asElement` makes sense for ElementBuilder but not for TypedBuilder. Although I admit that take decisions for "future" things are frequently a bad idea.